styleguide/derek/pattern-components/video/_video-embed.scss

```diff
@@ -9,14 +9,10 @@
 }
 
 .videoEmbed-wrapper {
-  margin-bottom: 10px;
+  margin-bottom: 40px;
   position: relative;
 }
 
-.embed-responsive {
```

Comment:
take it this was not being used anywhere? I'm just leaving comments to prove I looked at these things okay!